src/app/shared/post-it-collections/add-button/post-it-collection-add-button.component.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Lumeer: Modern Data Definition and Processing Platform
+ *
+ * Copyright (C) since 2017 Answer Institute, s.r.o. and/or its affiliates.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import {Component, EventEmitter, Input, Output} from '@angular/core';
+import {CorrelationIdGenerator} from '../../../core/store/correlation-id.generator';
+import {DEFAULT_COLOR, DEFAULT_ICON} from '../../../core/constants';
+import {CollectionModel} from '../../../core/store/collections/collection.model';
+
+@Component({
+  selector: 'post-it-collection-add-button',
+  templateUrl: './post-it-collection-add-button.component.html',
+  styleUrls: ['./post-it-collection-add-button.component.scss']
+})
+export class PostItCollectionAddButtonComponent {
+
+  @Output()
+  public clicked = new EventEmitter<CollectionModel>();
+
+  @Input()
+  public disabled: boolean;
+
+  public add() {
+    const newCollection = {
+      ...this.emptyCollection(),
+      correlationId: CorrelationIdGenerator.generate()
```

Comment:
It's totally up to you but much cleaner design would to have a generic add button, emit empty `click` event and react to it in the component where it is used.